src/Appwrite/Utopia/Response.php

```diff
@@ -238,8 +255,8 @@ class Response extends SwooleResponse
     public const MODEL_WEBHOOK_LIST = 'webhookList';
     public const MODEL_KEY = 'key';
     public const MODEL_KEY_LIST = 'keyList';
-    public const MODEL_PROVIDER = 'provider';
-    public const MODEL_PROVIDER_LIST = 'providerList';
+    public const MODEL_AUTH_PROVIDER = 'authProvider';
```

Comment:
How hard will it be for us to use same provider object for both auth and messaging in the future?